templates/digital-asset-template/move/Move.toml

```diff
@@ -1,9 +1,19 @@
 [package]
-name = 'my_app'
-version = '1.0.0'
-[dependencies.AptosFramework]
-git = 'https://github.com/aptos-labs/aptos-core.git'
-rev = 'main'
-subdir = 'aptos-move/framework/aptos-framework'
+name = "digital_asset_launchpad"
+version = "1.0.0"
+authors = []
+
 [addresses]
-module_addr='_'
\ No newline at end of file
+launchpad_addr = "_"
+minter = "_"
+
+[dev-addresses]
```

Comment:
I found this annoying when tryin to run different move commnads (npm run move:compile/publish) - was trying to fix it on the sdk side to accept optional params like `--dev` but then it introduces some other errors because of this field - thoughts on just removing it for now?

Reply:
removed